composables/drop/massmint/useDropMassMintListing.ts

```diff
@@ -44,6 +49,14 @@ export default () => {
         },
       )
     })
+
+    const watcherStop = useModalIsOpenTracker({
+      isOpen: computed(() => preferencesStore.listingCartModalOpen),
+      onChange: () => {
+        cartMiniDisabled.value = false
+        watcherStop()
+      },
```

Comment:
Is it a problem to stop the watcher? If the user makes a second time to list items, the watcher will stop and meanwhile show the `cartMini` component, right?

Reply:
> Is it a problem to stop the watcher? If the user makes a second time to list items, the watcher will stop and meanwhile show the cartMini component, right?

once the list modal is closed the cart mini is not disabled anymore , it's only for the drop mint flow 

if the users mints agains and and lists again the cart mini will be disabled again before opening the list modal and enabled once the listing modal is closed

if the user selects a nft from the drop gallery and opens the modal the cart should not be hidden( how it's working atm, same for profile page or explore page)

let me know if it makes sense